frontend/src/components/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,13 +1,11 @@
 import { FC, UserSignInRequestDto } from 'common/types/types';
 import { AppRoute } from 'common/enums/enums';
 import { useAppForm } from 'hooks/hooks';
-import { getNameOf, getValidClasses } from 'helpers/helpers';
+import { getNameOf } from 'helpers/helpers';
 import { userSignIn as userSignInValidationSchema } from 'validation-schemas/validation-schemas';
 import { Button, Input, Link } from 'components/common/common';
 import { DEFAULT_SIGN_IN_PAYLOAD } from './common';
-import authImage from 'assets/img/auth.png';
-import logo from 'assets/img/logo.svg';
-import styles from './styles.module.scss';
+import styles from 'components/auth/components/styles.module.scss';
```

Comment:
```suggestion
import styles from './styles.module.scss';
```
you need to use own styles